README.md

```diff
@@ -5,16 +5,12 @@ XML sitemap generator for Pimcore. Built from the document tree.
 ## Installation
 1. Run the command ```composer require byng/pimcore-sitemap-plugin```
 2. Enable the plugin from Extras => Extensions on Pimcore admin panel 
+3. Click install to set up ```sitemap_exclude``` property 
```

Comment:
Just a heads up on these, the three backticks are generally used for multi-line sections of code, like:

'''
<?php

class Foo extends Bar {
'''

(Replacing the ''' with ``` of course).

For in-line code, `like this here`, you just have to do one backtick either side, like 'hello', but replacing ' with `.